lib/exercises.js

```diff
@@ -1,17 +1,41 @@
 // This method will return an array of arrays.
 // Each subarray will have strings which are anagrams of each other
-// Time Complexity: ?
-// Space Complexity: ?
+// Time Complexity: O(n log n)
+// Space Complexity: O(n)
 function grouped_anagrams(strings) {
```

Comment:
Your time complexity is O(n + k log k) where k is the length of the longest word, or O(n) if you assume each word is small in size.